test_1:
  initial_pos_ugv_x: 54
  initial_pos_ugv_y: -15.7
  initial_pos_ugv_z: 0.8
  initial_pos_ugv_roll: 0.0
  initial_pos_ugv_pitch: 0.0
  initial_pos_ugv_yaw: 0.0
  initial_pos_uav_x: 51
  initial_pos_uav_y: -13.0
  initial_pos_uav_z: 2.8
  initial_pos_uav_roll: 0.0
  initial_pos_uav_pitch: 0.0
  initial_pos_uav_yaw: 0.0

